mnefun/_mnefun.py

```diff
@@ -842,6 +872,42 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
     run_subprocess(cmd)
 
 
+def run_sss_localy(p, subjects, run_indices):
+
+    cal_file = op.join(op.dirname(op.dirname(__file__)), 'sss_cal.dat')
+    ct_file = op.join(op.dirname(op.dirname(__file__)), 'ct_sparse.fif')
+    if p.st_duration is not None:
+        st_duration = float(p.st_duration)
```

Comment:
I don't think you need to do this -- mne-python should convert it to float if it's not None already.